pkg/placement/placement.go

```diff
@@ -234,12 +236,12 @@ func (p *Service) ReportDaprStatus(stream placementv1pb.Placement_ReportDaprStat
 
 	clientID, err := p.validateClient(stream)
 	if err != nil {
-		return err
+		return apiErrors.PlacementServiceInternalError(fmt.Sprintf("Internal Error: %s", codes.Internal))
```

Comment:
Same comment as below, please propagate the original errors returned in `p.validateClient`